server.js

```diff
@@ -1,248 +1,229 @@
+    console.table('\n', res, '\n');
+    startApp();
+  })
+};
+
+function addADepartment() {
+  inquirer.prompt([
+    {
+      name: 'newDept',
+      type: 'input',
+      message: 'What is the name of the department you want to add?'
+    }
+  ]).then((response) => {
+    connection.query(`INSERT INTO department SET ?`,
+      {
+        department_name: response.newDept,
+      },
+      (err, res) => {
         if (err) throw err;
-        console.table('\n', res, '\n');
+        console.log(`\n ${response.newDept} successfully added to database! \n`);
         startApp();
-    })
+      })
+  })
 };
 
-addADepartment = () => {
+function addARole() {
+  connection.query(`SELECT * FROM department;`, (err, res) => {
+    let departments = res.map(department => ({ name: department.department_name, value: department.department_id }));
     inquirer.prompt([
-        {
-        name: 'newDept',
+      {
+        name: 'title',
         type: 'input',
-        message: 'What is the name of the department you want to add?'   
-        }
+        message: 'What is the name of the role you want to add?'
+      },
+      {
+        name: 'salary',
+        type: 'input',
+        message: 'What is the salary of the role you want to add?'
+      },
+      {
+        name: 'deptName',
+        type: 'rawlist',
+        message: 'Which department do you want to add the new role to?',
+        choices: departments
+      },
     ]).then((response) => {
-        connection.query(`INSERT INTO department SET ?`, 
+      connection.query(`INSERT INTO role SET ?`,
         {
-            department_name: response.newDept,
+          title: response.title,
+          salary: response.salary,
+          department_id: response.deptName,
         },
         (err, res) => {
-            if (err) throw err;
-            console.log(`\n ${response.newDept} successfully added to database! \n`);
-            startApp();
+          if (err) throw err;
+          console.log(`\n ${response.title} successfully added to database! \n`);
+          startApp();
         })
     })
+  })
 };
 
-addARole = () => {
+function addAnEmployee() {
+  connection.query(`SELECT * FROM role;`, (err, res) => {
+    if (err) throw err;
+    let roles = res.map(role => ({ name: role.title, value: role.role_id }));
     connection.query(`SELECT * FROM department;`, (err, res) => {
-        let departments = res.map(department => ({name: department.department_name, value: department.department_id }));
+      if (err) throw err;
+      let departments = res.map(department => ({ name: department.department_name, value: department.department_id }));
+      connection.query(`SELECT * FROM employee;`, (err, res) => {
+        if (err) throw err;
+        let employees = res.map(employee => ({ name: employee.first_name + ' ' + employee.last_name, value: employee.employee_id }));
         inquirer.prompt([
+          createPrompt('firstName', 'input', "What is the new employee's first name?"),
```

Comment:
@samrogers15 You're creating a ton of these prompts and the repetition is both tedious and takes up a lot of room, so a helper function to generate them seemed prudent